### DISCUSSION TITLE
Multiple stations re-streaming one live

### DISCUSSION THREAD

**Discussion by DEMedus** [savonet/liquidsoap#1802] [Q&A] [answered]
I have an AzuraCast station, icecast, and so do a few of my podcasting buddies. We do live shows regularly and and are getting a schedule together so we don't overlap. 
I would like to have a way to schedule the start of a live re-stream of whoever is live, then pop back to the regular playlist when they disconnect. 
I can get it working to connect and re-stream with the schedule as I know when each show starts, but the shows sometimes go far over the scheduled time and I don't want my stream to cut back to my playlist in the middle of their show.
So does anyone know how to get the autoDJ or liquidsoap to check another stream to see if it is still live and switch back to the playlist if it isn't? Can this be done somehow, webhooks or some other way?

**Comment by toots** [comment, marked as answer]
Hi,

If you are using liquidsoap 2.0 and all the sources are in the same script, sources now have a `is_ready` method attached to them that I believe you should be able to use to programmatically figure out if a given source is ready..

**Reply by DEMedus** [reply]
Thank you I will dig in to this this weekend. I'm still new to all of this and am having a great time learning.
